app/src/main/java/com/example/wearit/ui/FavoritesScreen.kt

```diff
@@ -0,0 +1,328 @@
+                    .fillMaxWidth()
+                    .weight(1f),
+                icon = R.drawable.editing,
+                text = "EDIT",
+            )
+        }
+        LazyColumn(
+            modifier = Modifier.fillMaxWidth(),
+            contentPadding = PaddingValues(horizontal = 35.dp, vertical = 100.dp),
+            verticalArrangement = Arrangement.spacedBy(50.dp),
+        ) {
+            items(outfits) { outfit ->
+                Box(
+                    contentAlignment = Alignment.Center,
+                    modifier = Modifier
+                        .padding(5.dp, 5.dp)
+                        .clip(shape = RoundedCornerShape(50.dp))
+                        .fillMaxWidth()
+                        .border(
+                            width = 5.dp,
+                            color = MaterialTheme.colors.primary.copy(alpha = LocalContentAlpha.current),
+                            shape = RoundedCornerShape(50.dp)
+                        )
+
+                ) {
+
+                    MakeHorizontalPager(
+                        outfit.itemsInOutfit,
+                        getItemById,
+                        getItemPhotoByPhotoFilename
+                    );
+
+                }
+            }
+        }
+
+    }
+
+
+}
+
+
+@OptIn(ExperimentalPagerApi::class)
+@Composable
+fun MakeHorizontalPager(
+    itemsInOutfit: List<Int>,
+    getItemById: (Int) -> Item?,
+    getItemPhotoByPhotoFilename: (itemId: String) -> Bitmap
+
+) {
+    val pagerState = rememberPagerState(initialPage = 0)
+    val coroutineScope = rememberCoroutineScope()
+
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        HorizontalPager(
+            count = itemsInOutfit.size,
+            state = pagerState,
+            modifier = Modifier
+                .fillMaxWidth()
+        ) { currentPage ->
+            Row {
+                val finalItem = getItemById(itemsInOutfit[currentPage])
+                Image(
+                    bitmap = getItemPhotoByPhotoFilename(finalItem!!.photoFilename).asImageBitmap(),
+                    contentDescription = finalItem.name,
+                    modifier = Modifier
+                        .size(width = 200.dp, height = 340.dp)
+                )
+
+            }
+
+        }
+        Box(
+            contentAlignment = Alignment.BottomCenter,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 10.dp, bottom = 15.dp, end = 10.dp)
+        ) {
+            PagerIndicator(pagerState = pagerState, indicatorCount = itemsInOutfit.size) {
+                coroutineScope.launch {
+                    pagerState.scrollToPage(it)
+                }
+            }
+        }
+    }
+
+
+}
+
+@Composable
+fun BottomBar(
+    goToPickerScreen: () -> Unit,
+    goToWardrobe: () -> Unit,
+) {
+    Column(modifier = Modifier.background(color = Color.White)) {
```

Comment:
Usuń całą tę kolumnę (zostaw tylko Row), bo nie jest potrzebna. A kolor ustaw na tym Row'ie, ale nie White tylko MaterialTheme.color.background, żeby działało również w dark mode. Jakby pojawiały się jakieś problemy z tłem za przyciskami, to zmień kolejność modifier'ów